internal/assets/token_approval.go

```diff
@@ -151,13 +151,6 @@ func (am *assetManager) validateApproval(ctx context.Context, ns string, approva
 		}
 		approval.Pool = pool
 	}
-	if approval.Key == "" {
-		org, err := am.identity.GetLocalOrganization(ctx)
-		if err != nil {
-			return err
-		}
-		approval.Key = org.Identity
-	}
-
-	return nil
+	approval.Key, err = am.identity.ResolveInputSigningKeyOnly(ctx, approval.Key)
```

Comment:
This change is made across all the token connectors, which were actually not previously doing the resolve function on the key.

... which makes me realize the keys are not guaranteed to be resolvable by the blockchain connector in this case. So I need to tweak this before we merge, so they use a function that only does the default-setting, and does **not** do the resolve against the blockchain plugin.

Reply:
ok - I've addressed this.
Rather than switching address resolution off completely (which would be a big pain for the 90% case of Eth tokens + Eth blockchain), I've added a setting (default true) that allows it to be switched off for asset manager scenarios.

This might need to evolve in the future, if a FireFly use case requires advanced Ethereum resolution of keys (with things like HD wallet alias support etc.), in a mixed blockchain (Fabric core, Eth tokens) environment. I suggest the community addresses that issue if/when encountered.